arch/arm64/src/common/arm64_arch_timer.c

```diff
@@ -252,10 +253,11 @@ static int arm64_tick_start(struct oneshot_lowerhalf_s *lower,
   priv->callback = callback;
   priv->arg = arg;
 
-  /* Set the timeout */
+  next_cycle =
```

Comment:
No. I changed the code in current master, which doesn't incorporate such changes.

If you use SMP, please add the needed SMP related code after at least the single core tick timer work as it is supposed to.

Reply:
here is the patch: https://github.com/apache/nuttx/pull/13740

Reply:
Looks fine to me; although I don't see how it would affect to the issue mentioned in this PR; and I don't believe that this PR even conflicts with that one.

Please do recognize that there is a serious issue existing with the tick timer currently, and that I have provided both a way to re-produce it in qemu and real HW (see the discussion above) and also the correction for it in this PR.